import_data_into_db.py

```diff
@@ -15,6 +15,11 @@
 
     for row in anime_table:
 
+        if row['episodes'] == 'Unknown':
+            row['episodes'] = None
+        else:
```

Comment:
I would not put the "else" condition there.
But that's just a style preference so do what you think makes more sense